dev/com.ibm.ws.messaging.comms.server/src/com/ibm/ws/jfap/inbound/channel/CommsServerServiceFacade.java

```diff
@@ -106,16 +121,30 @@ public void activate(Map<String, Object> properties, ComponentContext context) {
         //Go ahead and Register JFAPChannel with Channel Framework by providing JFAPServerInboundChannelFactory
         _chfw_bunlde.getFramework().registerFactory("JFAPChannel", JFAPServerInboundChannelFactory.class);
 
-        inboundChain.init(endpointName, getCHFWBundle());
-        inboundSecureChain.init(endpointName + "-ssl", getCHFWBundle());
-
+        _useNettyTransport = ProductInfo.getBetaEdition() ? 
+				MetatypeUtils.parseBoolean(Inbound_ConfigAlias, USE_NETTY, properties.get(USE_NETTY), true):
+				false;
+
+        if(useNetty()) {
+        	inboundChain = new NettyInboundChain(this, false);
+        	inboundSecureChain = new NettyInboundChain(this, true);
+        	((NettyInboundChain)inboundChain).init(endpointName, getNettyBundle());
```

Comment:
It is not clear why the inits need to be in the if statements (over a bundle = useNetty?getNettyBundle:getCHFWBundle) and why casting is needed if init is an instance method
(perhaps my understanding is flawed though)